appveyor.yml

```diff
@@ -0,0 +1,33 @@
+# https://www.appveyor.com/docs/appveyor-yml
+
+version: "{build}"
+
+clone_depth: 10
+
+environment:
+  PLATFORM: "x64"
+  matrix:
+    - NODEJS_VERSION: "10"
+    - NODEJS_VERSION: "8"
+    - NODEJS_VERSION: "6"
```

Comment:
Ok with only testing x64 or (node v6/latest node) since we just want to test windows compat vs. diffferences in node

Reply:
Node 8 is LTS release too, why not ? We have more confidence adding node 8 other than remove it.